operator/integration-tests/model-in-image/run-test.sh

```diff
@@ -54,6 +54,7 @@ function usage() {
     WORKDIR               : /tmp/\$USER/mii-sample-work-dir
     DOMAIN_NAMESPACE      : sample-domain1-ns
     MODEL_IMAGE_NAME      : model-in-image 
+    MODEL_IMAGE_TAG       : ${WDT_DOMAIN_TYPE}-v2
```

Comment:
Also I think it'd be more clear if this was called MODEL_IMAGE_TAG_OVERRIDE and used to override MODEL_IMAGE_TAG where-ever it was set internally. (E.g. leave MODEL_IMAGE_TAG as an internal variable.)

Reply:
If MODEL_IMAGE_TAG is not set, it will use the default value.

Reply:
The point is that the usage help is incorrect.  The default changes based on use case (version, whether or not aux is involved, and whether WLS or JRF are involved).  The default is _not_ ${WDT_DOMAIN_TYPE}-v2 except for specific use cases (specific combinations of command-line and env var settings).